cnv/cnv_users_guide/cnv-installing-qemu-guest-agent.adoc

```diff
@@ -4,17 +4,30 @@ include::modules/cnv-document-attributes.adoc[]
 :context: cnv-installing-qemu-guest-agent
 toc::[]
 
+.Prerequisites
+
+* Verify that the guest agent is installed and running by entering the following command:
++
+----
+'systemctl status qemu-guest-agent'
+----
++
+
 The QEMU guest agent is a daemon that runs on the virtual machine. The agent
 passes network information on the virtual machine, notably the IP address of
 additional networks, to the host.
 
 include::modules/cnv-installing-qemu-guest-agent-on-linux-vm.adoc[leveloffset=+1]
 
-You can also install and start the QEMU guest agent using the
-*cloud-init*:*Use custom script* field of the wizard when creating either
+You can also install and start the QEMU guest agent by using the
+*custom script* field in the *cloud-init* section of the wizard when creating either
 virtual machines or virtual machines templates in the web console.
 
-For Windows virtual machines, the QEMU guest agent is included in the
-VirtIO drivers, which can be installed on
-xref:../../cnv/cnv_users_guide/cnv-installing-virtio-drivers-on-existing-windows-vm.adoc#cnv-installing-virtio-drivers-on-existing-windows-vm[an existing Windows virtual machine] or
-xref:../../cnv/cnv_users_guide/cnv-installing-virtio-drivers-on-new-windows-vmi#cnv-installing-virtio-drivers-on-new-windows-vm[during the installation of Windows on the virtual machine].
+include::modules/cnv-installing-qemu-guest-agent-on-windows-vm.adoc[leveloffset=+1]
+
+* Installing VirtIO drivers on an existing Windows virtual machine
```

Comment:
Why do you have a list in the middle of your assembly?

Reply:
Fixed.